code/ModelAdmin.php

```diff
@@ -183,6 +185,34 @@ public function Link($action = null)
         return parent::Link($action);
     }
 
+    /**
+     * Get the link for a modelTab on this ModelAdmin.
+     * @throws InvalidArgumentException if $modelTab is not managed by this ModelAdmin class.
+     */
+    public function getLinkForModelTab(string $modelTab): string
+    {
+        if (!array_key_exists($modelTab, $this->getManagedModels())) {
+            throw new InvalidArgumentException("$modelTab doesn't exist on this ModelAdmin.");
+        }
+        return parent::Link($this->sanitiseClassName($modelTab));
+    }
+
+    /**
+     * Get the link for editing an object inside this ModelAdmin.
+     */
+    public function getEditLinkForObject(DataObject $obj): string
```

Comment:
Should change this to `getEditLinkForDataObject()`

Reply:
I don't think that's a necessary change given it has the typehint to clearly indicate what should be passed in - but ultimately it makes little difference so I'll make this change.

Reply:
Done